ui/components/__tests__/__snapshots__/Footer.test.tsx.snap

```diff
@@ -0,0 +1,122 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Footer snapshots default 1`] = `
```

Comment:
I would expect to see the version appearing in this snapshot. Let me know if you want some help on testing sanpshops + hooks together.